SlackPrNotification.js

```diff
@@ -29,7 +29,7 @@ if (makePretty) {
                         block_id: "commit_title",
                         text: {
                             type: "mrkdwn",
-                            text: "*<" + prUrl + "|" + prTitle + ">* #" + prNum + " from *" + baseBranchText + "* to *" + compareBranchText + "*." + sendHereMention
+                            text: "[*#" + prNum + "*] *" + prTitle + "* from *" + baseBranchName + "* to *" + compareBranchName + "*." + sendHereMention
```

Comment:
Please correct the following.
- Replace from `baseBranchName` to `baseBranchText`.
- Replace from `compareBranchName` to `compareBranchText`.

Reply:
ooops.

I noticed these problems, and fixed them...in the other PR I just sent.

If the other PR is accepted, please close this one without accepting.